### ISSUE TITLE
AttributeError: 'OtherChoice' object has no attribute 'key'

### ISSUE THREAD

**Issue by benzo8** [bpoxy/plex-tvdb-alt-orderer#2]
Python: 3.8.10
OS: Linux mediaserver 5.4.0-150-generic #167-Ubuntu SMP Mon May 15 17:35:05 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
plex-tvdb-alt-orderer: plex-tvdb-alt-orderer-0.2.2

(Server name and Auth set as environment variables)

```
john@mediaserver ~ % plex-tvdb-alt-orderer
[?] Select the library to update: Documentaries
   Anime
 > Documentaries
   TV Shows
   YouTube

[?] Enter the name of the show: ?
Your TV show library doesn't contain a show with name '?'.

john@mediaserver ~ % plex-tvdb-alt-orderer
[?] Select the library to update: Documentaries
   Anime
 > Documentaries
   TV Shows
   YouTube

[?] Enter the name of the show: Grand
[?] Select the show to update: Grand Designs

Traceback (most recent call last):
  File "/home/john/.local/bin/plex-tvdb-alt-orderer", line 8, in <module>
    sys.exit(main())
  File "/home/john/.local/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/john/.local/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/john/.local/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/john/.local/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/john/.local/lib/python3.8/site-packages/plex_tvdb_alt_orderer/main.py", line 38, in main
    plex_show = get_plex_show(plex_section, plex_show_name)
  File "/home/john/.local/lib/python3.8/site-packages/plex_tvdb_alt_orderer/main.py", line 91, in get_plex_show
    return inquirer.prompt([inquirer.List("show", message="Select the show to update", choices=dict_to_tuple(shows_dict))])["show"]
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/prompt.py", line 11, in prompt
    answers[question.name] = render.render(question, answers)
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/render/console/__init__.py", line 38, in render
    return self._event_loop(render)
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/render/console/__init__.py", line 49, in _event_loop
    self._print_options(render)
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/render/console/__init__.py", line 66, in _print_options
    for message, symbol, color in render.get_options():
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/render/console/_list.py", line 51, in get_options
    symbol = "+" if choice == GLOBAL_OTHER_CHOICE else self.theme.List.selection_cursor
  File "/home/john/.local/lib/python3.8/site-packages/inquirer/questions.py", line 27, in __eq__
    return self.value == other
  File "/home/john/.local/lib/python3.8/site-packages/plexapi/base.py", line 458, in __eq__
    return other not in [None, []] and self.key == other.key
AttributeError: 'OtherChoice' object has no attribute 'key'
```

**Comment by bpoxy** [comment]
This is the result of using `plexapi.video.Show` instances as choice values when building the `inquirer.List` instance in `get_plex_show()`. Internally, `inquirer.prompt()` [compares each choice value to `inquirer.render.console._other.GLOBAL_OTHER_CHOICE`](https://github.com/magmax/python-inquirer/blob/042747e48e91039fa83f35a183c081201c4d397c/src/inquirer/render/console/_list.py#L51) (an instance of `inquirer.render.console._other.OtherChoice`). There is nothing inherently wrong with this but because `plexapi.PlexPartialObject` (a base class of `plexapi.video.Show`) [implements `==` in such a way that it expects the objects being compared to possess a `key` attribute](https://github.com/pkkid/python-plexapi/blob/e2587eca31b3dc6135dab78119516ab5ba7a7c6b/plexapi/base.py#L491-L492)), an exception is raised upon comparison against `inquirer.render.console._other.GLOBAL_OTHER_CHOICE` (which has no such attribute).

A dirty workaround is to execute `inquirer.render.console._other.GLOBAL_OTHER_CHOICE.key = None` before calling `inquirer.prompt()` but I'm not keen on that approach as it relies on knowledge about the internals of 2 different libraries. Therefore, I'm instead going to resolve this by only passing choice labels to `inquirer` and mapping the user's choice back to the corresponding object myself.

**Comment by bpoxy** [comment]
Fixed in v0.2.3.
